app/controllers/posts_controller.rb

```diff
@@ -50,6 +50,21 @@ def interactions
     end
   end
 
+  def mentionable_in_comment
+    respond_to do |format|
+      format.json {
+        if params[:id].present? && params[:q].present?
```

Comment:
We can do this without making a query, by reusing data we already have in the frontend. This could be more complex for the frontend, but will simplify and unload the backend.

Maybe we should ask @AugierLe42e for his opinion here?